streams/test-utils/src/main/java/org/apache/kafka/streams/InternalTopologyBuilderAccessor.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.streams;
```

Comment:
This is a meta comment: should we consider using a sub-package name than o.a.k.streams? Perhaps `o.a.k.streams.testutils`?

Reply:
For this specific class, we need it to put it in this package, because the InternalTopologyBuilder is accessible package private only and I don't think we should make it public.